test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -381,14 +384,9 @@ async function setupLoopbackFallback(t, widgetOptions) {
   return deviceChallengeFalllbackPage;
 }
 
-const skipTest = () => {
-  return null;
-  // return test.skip;
-};
-
-// TODO: fix quarantined test - OKTA-645716
-skipTest()
-  ?.requestHooks(loopbackSuccessLogger, loopbackSuccessMock)?.('in loopback server approach, probing and polling requests are sent and responded', async t => {
+test
+  .requestHooks(loopbackSuccessLogger, loopbackSuccessMock)('in loopback server approach, probing and polling requests are sent and responded', async t => {
+    failureCount = 0;
```

Comment:
Need to reset dynamic global vars in order to let test retry in case of failing

Reply:
Could these counter resets be handled by the `setup` function?